### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by saadahmadhere**

### CODE REVIEW

File: src/Components/Filters/Filters.jsx [hunk 1]
```diff
@@ -57,25 +57,27 @@ const Filters = () => {
...
-            Earphones
+          <label>
+            <input
+              type="checkbox"
+              name="category"
+              id="earphones"
+              onChange={(e) =>
+                filterDispatch({
+                  type: "CATEGORY",
+                  payload: "ear phones",
```
Comment (abhij1607) [inline src/Components/Filters/Filters.jsx:68]: this seems different from the CategoryName spelling, cross-check it once

File: src/backend/db/categories.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ import { v4 as uuid } from "uuid";
 export const categories = [
   {
     _id: uuid(),
-    categoryName: "Earphones",
+    categoryName: "Ear Phones",
```
Comment (abhij1607) [inline src/backend/db/categories.js:11]: cross-check this with above comment

**Review by abhij1607 — COMMENTED** [review]
Neat & clean code || Good naming practices followed

left a comment, cross-check it once

**Review by Tanay-J — APPROVED** [review]
The changes are to fix issues in earlier code, hence most of the code is already reviewed.
Please check for the suggestions by other reviewer and you are good to go.

**Review by MaruthiPothuganti — COMMENTED** [review]
Code refactoring done good. Got to learn How useNavigate works. Good to go.

File: src/Components/Filters/Filters.jsx (hunk 1 above)
Comment (saadahmadhere) [reply src/Components/Filters/Filters.jsx:68]: I have used toLowerCase() later on. No worries.

Comment (saadahmadhere) [reply src/Components/Filters/Filters.jsx:68]: I have used toLowerCase() later on. No worries.

File: src/backend/db/categories.js (hunk 2 above)
Comment (saadahmadhere) [reply src/backend/db/categories.js:11]: done

**Pull request merged**